docs/UserGuide.md

```diff
@@ -426,6 +430,115 @@ Invalid Input Example | Application Output
 
 
 ## Payments
+
+CampusConnect allows you to keep track of payments between you and your friends! Each contact has a **balance**, which is the amount of money that you owe them (or that they owe you). This balance is positive if they owe you money, and negative if you owe them money.
+
+Every time you record a payment to or from a contact, their balance will be updated accordingly so that you can settle your debts easily. Instead of displaying positive or negative balances, **balances are reflected directly in the contact list** with easily-understandable human-readable explanations, like so:
+
+![Payment Image](images/ContactArrowPayment.png)
```

Comment:
the arrows seem a little fuzzy when rendered, not sure if you are using a solid arrow in labelling of picture

Reply:
Oh yeah I used a fuzzy arrow, is that ok?